File: 1446-AngleBetweenHandsOfAClock/1446-AngleBetweenHandsOfAClock.java
// Last updated: 6/25/2026, 11:33:13 PM
class Solution {
    public double angleClock(int hour, int minutes) {    
    double n;
    n=Math.abs((hour*30)-(minutes*5.5));
    double n3n=(360-n);

     return Math.min(n3n,n);

    }
}

File: 1446-AngleBetweenHandsOfAClock/1446-AngleBetweenHandsOfAClock.cpp
// Last updated: 7/7/2026, 10:25:10 PM
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double hourangle= (hour%12)*30 +minutes*0.5;
        double minuteangle= minutes*6;
        double diff= abs(hourangle- minuteangle);
        return min(diff, 360-diff);
    }
};